packages/server/src/automations/triggers.ts

```diff
@@ -140,7 +141,12 @@ function rowPassesFilters(row: Row, filters: SearchFilters) {
 
 export async function externalTrigger(
   automation: Automation,
-  params: { fields: Record<string, any>; timeout?: number; appId?: string },
+  params: {
+    fields: Record<string, any>
+    timeout?: number
+    appId?: string
+    user?: UserBindings | undefined
```

Comment:
Why does `undefined` need specified here as well as the optional `user?:` - only one should be needed.

Reply:
Yea, I can just remove the undefined. It was just a leftover from a previous type I had. No longer needed